code/game/objects/structures/door_assembly.dm

```diff
@@ -21,11 +22,27 @@
 	var/material_type = /obj/item/stack/sheet/iron
 	var/material_amt = 4
 
+/obj/structure/door_assembly/multi_tile
+	dir = EAST
+	/// Whether or not the assembly is for a multi-tile airlock.
+	multi_tile = TRUE
+
 /obj/structure/door_assembly/Initialize(mapload)
 	. = ..()
 	update_appearance()
 	update_name()
 
+/obj/structure/door_assembly/multi_tile/Initialize(mapload)
+	. = ..()
+	multi_tile = get_size_in_tiles(src) > 1
```

Comment:
Why do we bother initialising it as true in this case?

Reply:
Should be fine as it's part of the subtype